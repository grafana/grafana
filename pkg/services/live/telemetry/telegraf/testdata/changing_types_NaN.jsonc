//  ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ
//  
//  Frame[0] 
//  Name: system
//  Dimensions: 5 Fields by 4 Rows
//  +----------------+-------------------------------+------------------+------------------+-----------------+
//  | Name: labels   | Name: time                    | Name: sensor     | Name: sensor2    | Name: state     |
//  | Labels:        | Labels:                       | Labels:          | Labels:          | Labels:         |
//  | Type: []string | Type: []time.Time             | Type: []*float64 | Type: []*float64 | Type: []*string |
//  +----------------+-------------------------------+------------------+------------------+-----------------+
//  | host=A         | 2021-03-22 08:51:30 +0000 UTC | 0                | null             | aaa             |
//  | host=B         | 2021-03-22 08:51:30 +0000 UTC | null             | 0                | bbb             |
//  | host=A         | 2021-03-22 08:51:31 +0000 UTC | null             | 0                | ccc             |
//  | host=B         | 2021-03-22 08:51:31 +0000 UTC | 0                | null             | 1               |
//  +----------------+-------------------------------+------------------+------------------+-----------------+
//  
//  
//  ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ
{
  "status": 200,
  "frames": [
    {
      "schema": {
        "name": "system",
        "fields": [
          {
            "name": "labels",
            "type": "string",
            "typeInfo": {
              "frame": "string"
            }
          },
          {
            "name": "time",
            "type": "time",
            "typeInfo": {
              "frame": "time.Time"
            }
          },
          {
            "name": "sensor",
            "type": "number",
            "typeInfo": {
              "frame": "float64",
              "nullable": true
            }
          },
          {
            "name": "sensor2",
            "type": "number",
            "typeInfo": {
              "frame": "float64",
              "nullable": true
            }
          },
          {
            "name": "state",
            "type": "string",
            "typeInfo": {
              "frame": "string",
              "nullable": true
            }
          }
        ]
      },
      "data": {
        "values": [
          [
            "host=A",
            "host=B",
            "host=A",
            "host=B"
          ],
          [
            1616403090000,
            1616403090000,
            1616403091000,
            1616403091000
          ],
          [
            0,
            null,
            null,
            0
          ],
          [
            null,
            0,
            0,
            null
          ],
          [
            "aaa",
            "bbb",
            "ccc",
            "1"
          ]
        ]
      }
    }
  ]
}
