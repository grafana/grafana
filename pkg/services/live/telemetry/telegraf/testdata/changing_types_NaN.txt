system,host=A sensor=0,sensor2="NaN",state="aaa" 1616403090000000000
system,host=B sensor="NaN",sensor2=0,state="bbb" 1616403090000000000
system,host=A sensor="NaN",sensor2=0,state="ccc" 1616403091000000000
system,host=B sensor=0,sensor2="NaN",state=1 1616403091000000000
