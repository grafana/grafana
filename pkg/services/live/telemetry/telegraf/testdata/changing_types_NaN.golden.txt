ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] 
Name: system
Dimensions: 5 Fields by 4 Rows
+----------------+-------------------------------+------------------+------------------+-----------------+
| Name: labels   | Name: time                    | Name: sensor     | Name: sensor2    | Name: state     |
| Labels:        | Labels:                       | Labels:          | Labels:          | Labels:         |
| Type: []string | Type: []time.Time             | Type: []*float64 | Type: []*float64 | Type: []*string |
+----------------+-------------------------------+------------------+------------------+-----------------+
| host=A         | 2021-03-22 11:51:30 +0300 MSK | 0                | null             | aaa             |
| host=B         | 2021-03-22 11:51:30 +0300 MSK | null             | 0                | bbb             |
| host=A         | 2021-03-22 11:51:31 +0300 MSK | null             | 0                | ccc             |
| host=B         | 2021-03-22 11:51:31 +0300 MSK | 0                | null             | 1               |
+----------------+-------------------------------+------------------+------------------+-----------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////mAIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAFQAAAACAAAAKAAAAAQAAADs/f//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAAAz+//8IAAAAEAAAAAYAAABzeXN0ZW0AAAQAAABuYW1lAAAAAAUAAACsAQAANAEAANQAAABgAAAABAAAAEr///8UAAAAPAAAADwAAAAAAAUBOAAAAAEAAAAEAAAAaP7//wgAAAAQAAAABQAAAHN0YXRlAAAABAAAAG5hbWUAAAAAAAAAAGD+//8FAAAAc3RhdGUAAACi////FAAAADwAAAA8AAAAAAADATwAAAABAAAABAAAAMD+//8IAAAAEAAAAAcAAABzZW5zb3IyAAQAAABuYW1lAAAAAAAAAAAy////AAACAAcAAABzZW5zb3IyAAAAEgAYABQAEwASAAwAAAAIAAQAEgAAABQAAAA8AAAAPAAAAAAAAwE8AAAAAQAAAAQAAAAw////CAAAABAAAAAGAAAAc2Vuc29yAAAEAAAAbmFtZQAAAAAAAAAAov///wAAAgAGAAAAc2Vuc29yAACe////FAAAADwAAABEAAAAAAAACkQAAAABAAAABAAAAIz///8IAAAAEAAAAAQAAAB0aW1lAAAAAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAMABAAAAHRpbWUAABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAARAAAAEgAAAAAAAAFRAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAABAAAAAGAAAAbGFiZWxzAAAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAAGAAAAbGFiZWxzAAD/////aAEAABQAAAAAAAAADAAWABQAEwAMAAQADAAAAMgAAAAAAAAAFAAAAAAAAAMDAAoAGAAMAAgABAAKAAAAFAAAANgAAAAEAAAAAAAAAAAAAAAMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGAAAAAAAAAAYAAAAAAAAABgAAAAAAAAAMAAAAAAAAAAAAAAAAAAAADAAAAAAAAAAIAAAAAAAAABQAAAAAAAAAAgAAAAAAAAAWAAAAAAAAAAgAAAAAAAAAHgAAAAAAAAACAAAAAAAAACAAAAAAAAAACAAAAAAAAAAoAAAAAAAAAAAAAAAAAAAAKAAAAAAAAAAGAAAAAAAAAC4AAAAAAAAABAAAAAAAAAAAAAAAAUAAAAEAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAgAAAAAAAAAEAAAAAAAAAAIAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAAGAAAADAAAABIAAAAYAAAAAAAAAGhvc3Q9QWhvc3Q9Qmhvc3Q9QWhvc3Q9QgA0S9kLnm4WADRL2QuebhYA/uUUDJ5uFgD+5RQMnm4WCQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAwAAAAYAAAAJAAAACgAAAAAAAABhYWFiYmJjY2MxAAAAAAAAEAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADwAAAAAAAMAAQAAAKgCAAAAAAAAcAEAAAAAAADIAAAAAAAAAAAAAAAAAAAAAAAAAAAACgAMAAAACAAEAAoAAAAIAAAAVAAAAAIAAAAoAAAABAAAAOz9//8IAAAADAAAAAAAAAAAAAAABQAAAHJlZklkAAAADP7//wgAAAAQAAAABgAAAHN5c3RlbQAABAAAAG5hbWUAAAAABQAAAKwBAAA0AQAA1AAAAGAAAAAEAAAASv///xQAAAA8AAAAPAAAAAAABQE4AAAAAQAAAAQAAABo/v//CAAAABAAAAAFAAAAc3RhdGUAAAAEAAAAbmFtZQAAAAAAAAAAYP7//wUAAABzdGF0ZQAAAKL///8UAAAAPAAAADwAAAAAAAMBPAAAAAEAAAAEAAAAwP7//wgAAAAQAAAABwAAAHNlbnNvcjIABAAAAG5hbWUAAAAAAAAAADL///8AAAIABwAAAHNlbnNvcjIAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAADwAAAA8AAAAAAADATwAAAABAAAABAAAADD///8IAAAAEAAAAAYAAABzZW5zb3IAAAQAAABuYW1lAAAAAAAAAACi////AAACAAYAAABzZW5zb3IAAJ7///8UAAAAPAAAAEQAAAAAAAAKRAAAAAEAAAAEAAAAjP///wgAAAAQAAAABAAAAHRpbWUAAAAABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAEAAAAdGltZQAAEgAYABQAAAATAAwAAAAIAAQAEgAAABQAAABEAAAASAAAAAAAAAVEAAAAAQAAAAwAAAAIAAwACAAEAAgAAAAIAAAAEAAAAAYAAABsYWJlbHMAAAQAAABuYW1lAAAAAAAAAAAEAAQABAAAAAYAAABsYWJlbHMAAMgCAABBUlJPVzE=
